ovos_PHAL_plugin_homeassistant/logic/socketclient.py

```diff
@@ -289,3 +290,35 @@ def call_service_sync(self, domain, service, service_data):
         task = asyncio.run_coroutine_threadsafe(
             self.call_service(domain, service, service_data), self.loop)
         return task.result()
+
```

Comment:
Are these worth keeping for reference or just remove them?

Reply:
I kept them for reference, and also because once Neon deprecates Python 3.7 they're valid for all supported versions of Python.